indexing-service/src/main/java/io/druid/indexing/common/actions/SegmentListUsedAction.java

```diff
@@ -53,9 +69,9 @@ public String getDataSource()
   }
 
   @JsonProperty
```

Comment:
Not preserving backwards compatibility could cause tasks to fail during upgrade. We should preserve the old field for one more release to allow for seamless upgrades like we always do.